RELEASES.md

```diff
@@ -1,3 +1,123 @@
+Version 1.61.0 (2022-05-19)
+==========================
+
+Language
+--------
+
+- [`const fn` signatures can now include generic trait bounds][93827]
+- [`const fn` signatures can now use `impl Trait` in argument and return position][93827]
+- [Function pointers can now be created, cast, and passed around in a `const fn`][93827]
+- [Recursive calls can now set the value of a function's opaque `impl Trait` return type][94081]
+
+Compiler
+--------
+
+- [Linking modifier syntax in `#[link]` attributes and on the command line, as well as the `whole-archive` modifier specifically, are now supported][93901]
+- [Update to LLVM 14.0.0][95247]
```

Comment:
Not sure that this is relnote-worthy -- we already mentioned LLVM 14 in 1.60, although it was pre-release. There was not much functional change in this 14.0.0 bump. As a test, will we also mention the bumps to 14.0.1 etc?

Reply:
Ah, I didn't notice that. Removing.